docs/tutorial/creating_a_tweet_fieldtype.md

```diff
@@ -26,7 +26,7 @@ This tutorial is aimed at developers who are familiar with eZ Platform and are c
 
 This tutorial will demonstrate how to create a Field Type on the example of a *Tweet* Field Type. It will:
 
-- Accept as input the URL of a tweet (https://twitter.com/&lt;username&gt;/status/&lt;id&gt;)
+- Accept as input the URL of a tweet (`https://twitter.com/<username>/status/<id>`)
```

Comment:
Minor improvement suggestion: we use curly brackets to signify placeholders in routes. Maybe we should use that for these examples as well ? `https://twitter.com/{username}/status/{id}`